config/200-source-observer-clusterrole.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use this aggregated ClusterRole when you need read "Sources".
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: source-observer
```

Comment:
This feels fairly permanent. We need to decide if this is `source-observer`, `importer-observer`, or something else.

In general, we need to ensure all of our user interfaces use a single word, not a mixture of the two.

Reply:
I still disagree with importer. until the api group and the duck are forcible renamed, I would like to keep them consistent.